x/delayedack/keeper/rollapp_packet.go

```diff
@@ -153,17 +168,38 @@ func (k Keeper) ListRollappPendingPackets(
 }
 
 func (k Keeper) GetAllRollappPackets(ctx sdk.Context) (list []commontypes.RollappPacket) {
-	store := prefix.NewStore(ctx.KVStore(k.storeKey), types.KeyPrefix(commontypes.RollappPacketKeyPrefix))
+	store := ctx.KVStore(k.storeKey)
 
 	// Iterate over the range from lastProofHeight to proofHeight
-	iterator := sdk.KVStorePrefixIterator(store, []byte{})
+	iterator := sdk.KVStorePrefixIterator(store, commontypes.AllRollappPacketKeyPrefix)
 	defer iterator.Close() // nolint: errcheck
 
 	for ; iterator.Valid(); iterator.Next() {
 		var val commontypes.RollappPacket
-		k.cdc.MustUnmarshal(iterator.Value(), &val)
+		err := k.cdc.Unmarshal(iterator.Value(), &val)
```

Comment:
why u don't want to panic here? corrupted store is blocker, isn't it?

Reply:
yea we can panic here. this method is only called from init genesis.